lib/src/deserializer.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// deserializer.dart
+// A deserialization block, deserializing narrow input data onto a wide channel.
+//
+// 2024 August 27
+// Author: desmond Kirkpatrick <desmond.a.kirkpatrick@intel.com>
+
+import 'package:meta/meta.dart';
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+
+/// Aggregates data from a serialized stream
+class Deserializer extends Module {
+  /// Clk input
+  @protected
+  Logic get clk => input('clk');
+
+  /// Reset input
+  @protected
+  Logic get reset => input('reset');
+
+  /// Run deserialization whenever [enable] is true
+  @protected
+  Logic? get enable => input('enable');
+
+  /// Serialized input, one data item per clock
+  Logic get serialized => input('serialized');
+
+  /// Aggregated data output
+  LogicArray get deserialized => output('deserialized') as LogicArray;
+
+  /// Valid out when data is reached
+  Logic get done => output('done');
+
+  /// Return the count as an output
```

Comment:
This doc comment could be improved to explain the meaning of the count (you explained it well in the markdown doc)

Reply:
I can add that.  Should all comments that preface classes and methods end in '.'?

Reply:
Yes, I think so.  I try to follow the Dart style guides, e.g.
https://dart.dev/effective-dart/documentation#do-format-comments-like-sentences